tests/basics_test.py

```diff
@@ -90,9 +90,9 @@ def test_const(self):
         """Test that modifying const Doodds is not allowed.
         """
         d = challenge.basics.Doodad.get_const()
-        self.assertIsInstance(d, challenge.basics.Doodad)
-        self.assertRaises(TypeError, setattr, d, "name", "k")
-        self.assertRaises(TypeError, setattr, d, "value", 9)
+        self.assertIsInstance(d, challenge.basics.ImmutableDoodad)
+        self.assertRaises(AttributeError, setattr, d, "name", "k")
+        self.assertRaises(AttributeError, setattr, d, "value", 9)
```

Comment:
Ok, I have changed this. Note that this does require even more test modification. But it might be nicer this way. Tell me what you think.